Model/Config.php

```diff
@@ -146,7 +146,9 @@ public function getConfig($configPath, $scopeId = null, $scope = null)
         if (!$scope && $this->isSingleStoreMode()) {
             return $this->scopeConfig->getValue($configPath);
         }
-        return $this->scopeConfig->getValue($configPath, $scope ?: ScopeInterface::SCOPE_STORE, is_null($scopeId) ? $this->storeManager->getStore()->getId() : $scopeId);
+        $storeId = ($storeId === null) ? $this->getCurrentStoreId() : $storeId;
```

Comment:
This would lead to an error. Should be:
`$storeId = ($scopeId === null) ? $this->getCurrentStoreId() : $scopeId;`

Reply:
@pniel-cohen , Fixed

Reply:
@pniel-cohen , it's  fixed.